2.TXT

```diff
@@ -0,0 +1,13 @@
+  INSERT INTO FOLKS (first_name, last_name, age)
+  VALUES ('Jey', 'Street', 36);
+ 
+  INSERT INTO FOLKS (last_name, first_name, age)
+  VALUES ('Silent','Bob', 41);
+   
+  INSERT INTO FOLKS (first_name, last_name)
+  VALUES ('Vadym','Marchenko');
+  
+  INSERT INTO FOLKS VALUES (4,'Doggo','Mexicanos', 3);

```

Comment:
It is not necessary to put `id` here since you've created this field with `AUTOINCREMENT`